docs/tutorials/getting_started.md

```diff
@@ -0,0 +1,135 @@
+# Getting started
+
+## Prerequisites
+
+1. Python installed on your system.
+2. Access to a microgrid system supported by the `frequenz.sdk` or you can use
+   the sandbox.
+3. Basic knowledge of microgrid concepts.
+4. Familiarity with
+   [Channels](https://frequenz-floss.github.io/frequenz-channels-python/latest/).
+
+## Set up your Frequenz development environment
+
+### Setting up a Python environment
+
+Before you begin, make sure you have a working Python environment. You can
+create a virtual environment to manage dependencies.
+
+```bash
+# Create a virtual environment (optional but recommended)
+python -m venv frequenz_sdk_env
+source frequenz_sdk_env/bin/activate
+# On Windows, use 'frequenz_sdk_env\Scripts\activate'
+```
+
+### Install the Frequenz SDK
+
+Install the Frequenz SDK. For this tutorial, this is all you need, but in the
+future, you might also need additional packages based on your project
+requirements.
+
+```bash
+pip install frequenz-sdk  # Install the Frequenz SDK
+```
+
+## Create a project
+
+### Create a Python file
+
+You can start by simply creating a Python script (e.g., `pv_optimization.py`)
+using your favorite text editor.
+
+### Use Frequenz repo config
+
+As an alternative and for larger projects, it's recommended to set up the
+project using the [Frequenz repo
+configuration](https://frequenz-floss.github.io/frequenz-repo-config-python/latest).
+
+## Import necessary modules
+
+You can now open the app's main file and start adding content. Begin by
+importing the necessary libraries.
+
+```python
+import asyncio
+import os
+
+from datetime import timedelta
+from frequenz.sdk import microgrid
+from frequenz.sdk.actor import ResamplerConfig
+```
+
+## Create the application skeleton
+
+The main logic of your application will run within an async function. Let's
+create a skeleton that contains all the necessary code to initialize a
+microgrid.
+
+```python
+async def run() -> None:
+    # Configuration from environment variables
+    microgrid_host = os.getenv(
+        "MICROGRID_HOST", default="microgrid.sandbox.api.frequenz.io"
+    )
+    microgrid_port = int(os.getenv("MICROGRID_PORT", default="62060"))
+
+    # Initialize the microgrid
+    await microgrid.initialize(
+        microgrid_host,
+        microgrid_port,
+        ResamplerConfig(resampling_period=timedelta(seconds=1)),
+    )
```

Comment:
I'm not sure all the ENV stuff is not too distracting for the tutorial, any reason not to hardcode the connections details here (in a python variable), and add a comment that normally you would use env vars for this? Having the ENV stuff adds complexity here, and then when running the app.

Reply:
Yeah I'm all for hard coding for simplicity.